### PR TITLE
feat(formula): add asin function

### LINKED ISSUE #2043: [Bug] When dragging a cell to a formula, the formula calculation result is incorrect.
### 在您提交此问题之前，您是否检查了以下内容？

- [X] 这真的是个问题吗？
- [X] 我已经在 Github Issues 中搜索过了，但没有找到类似的问题。

### 受影响的包和版本

dev

### 复现链接

1、A1=sum(B1+c1),拖拽有内容（“测试”）的单元格到A1,

### 预期行为

拖拽后单元格显示为“测试”

### 实际行为

单元格内容未改变
[2024_4_23 11_26_03 video.webm](https://github.com/dream-num/univer/assets/154207126/186d74c6-fa4a-46e6-a5fd-f4c3cf53b797)

### 运行环境

_No response_

### 系统信息

_No response_

### PR DESCRIPTION
**Pull request by Dushusir**
- [x] ASIN
- [x] ASINH
- [x] ATAN
- [x] ATAN2
- [x] ATANH
- [x] COS
- [x] TAN
- [x] TANH

close #2043(在其他PR修复)

## Pull Request Checklist

- [x] Related tickets or issues have been linked in the PR description (or missing issue).
- [x] [Naming convention](https://github.com/dream-num/univer/blob/dev/docs/NAMING_CONVENTION.md) is followed (**do please** check it especially when you created new plugins, commands and resources).
- [x] Unit tests have been added for the changes (if applicable).
- [x] Breaking changes have been documented (or no breaking changes introduced in this PR).

### CODE REVIEW

**Comment by oumomomo** [comment]
ATAN2函数说明语言不对
image

**Comment by Dushusir** [comment]
all fixed

**Comment by oumomomo** [comment]
cos函数
1、=cos（A1:B1）A1B1输入值均为2，结果位数不一致
image

**Comment by Dushusir** [comment]
> cos/tan/tanh函数都有下面的问题 =cos（A1:B1）A1B1输入值均为2，结果位数不一致 image

fixed

File: packages/sheets/src/controllers/number-cell.controller.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        this._initInterceptorCellContent();
+    }
+
+    private _initInterceptorCellContent() {
+        this.disposeWithMe(
+            this._sheetInterceptorService.intercept(INTERCEPTOR_POINT.CELL_CONTENT, {
+                priority: 11,
+                handler: (cell, location, next) => {
+                    // Dealing with precision issues
+                    if (cell?.t === CellValueType.NUMBER && typeof cell?.v === 'number') {
```
Comment (VicKun4937) [inline packages/sheets/src/controllers/number-cell.controller.ts:42]: 这个条件够不够判断这个单元格是数字，比如我值是个数字，但是我设置的format 是‘@’之类的，他其实应该被当成字符串的，或者yyyy-mm-dd 会被当成日期， 不清楚是不是在这里之前已经处理好了

Comment (Dushusir) [reply packages/sheets/src/controllers/number-cell.controller.ts:48]: 确实会有问题，我把包含数字格式的单元格跳过了

**Review by DR-Univer — APPROVED** [review]

**Pull request merged**